CircuitPython_WeatherCloud/code.py

```diff
@@ -156,9 +156,11 @@
                 next_bolt_time = time.monotonic() + random.randint(1, 5)
```

Comment:
The `has_sound = True` right above here inside the Thunderstorm block was left in but the others were removed.  https://github.com/justmobilize/Adafruit_Learning_System_Guides/blob/02d49a92e79d2bd1698485d35ec6efb59325bf9e/CircuitPython_WeatherCloud/code.py#L154

It doesn't seem like it would hurt anything to me, but I lean towards removing it since it doesn't seem to be needed any longer as it get sets below when the WaveFile is created.

Reply:
So this one is a little different as for thunder it randomly sets a filename below. I can clean up the logic a bit to make it more clear (just below there's a has_sound and not thunder)

Reply:
But this line sets `has_sound` to `True` for all types since it's in a block based on the `wave_filename` which all types do set.

So if I am reading it correctly `has_sound` gets set to true for all types  by the current logic, but set to `True` twice for the Thunderstorm.

Reply:
Since thunderstorm doesn't set `wave_filename` it doesn't get set again

Reply:
Ah, yep. Okay thank you. You can refactor it further if you want, but no need to if you don't. It's essentially got the same logic it started with, I misunderstood it originally but no need to change it really.